src/platform/qpg6100/args.gni

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+import("//build_overrides/qpg6100_sdk.gni")
+
+# ARM architecture flags will be set based on efr32_family.
+arm_platform_config = "${qpg6100_sdk_build_root}/qpg6100_arm.gni"
+
+mbedtls_target = "${qpg6100_sdk_build_root}:qpg6100_sdk"
+
+openthread_external_mbedtls = mbedtls_target
+
+chip_device_platform = "qpg6100"
+
+lwip_platform = "qpg6100"
+lwip_ipv6 = true
+lwip_ipv4 = false
+lwip_api = true
+
+chip_inet_config_enable_ipv4 = false
+chip_inet_config_enable_dns_resolver = false
+
+chip_build_tests = false
+
+# TODO - enable OpenThread build flow
```

Comment:
Add GH issue number for TODO.
Eventually remove commented out code (and include it in the GH issue for reference)

Reply:
https://github.com/project-chip/connectedhomeip/issues/2939 created